HW0/P6/P6B.py

```diff
@@ -16,17 +16,27 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [10**-6, 10**-5,10**-4,10**-3,10**-2,10**-1,1]
```

Comment:
I think you might want to have more data points, here is a good way to generate a lot of points (50 by default) 
np.logspace(-6, 0), and you can specify how many points you want as a third argument: np.logspace(-6, 0, 20) for 20 points